iothub/device/samples/Getting Started/InvokeDeviceMethod/InvokeDeviceMethod.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net6.0</TargetFramework>
+    <LangVersion>9.0</LangVersion>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.Devices" Version="1.38.1" />
```

Comment:
Nick/Nate, here's one that doesn't use a project reference and will not build until we do.